tools/backup-setup/config/backup.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ark.heptio.com/v1
+kind: Backup
+metadata:
+  name: kyma-backup
+  namespace: heptio-ark
+spec:
+  includedNamespaces:
+  - production
```

Comment:
Can you use a wildcard (*) here as this file should backup all namespaces which are not system namespaces.

Reply:
Yes we can, but actually the official documentation says:
```
# Array of namespaces to include in the backup. If unspecified, all namespaces are included.
# Optional. 
```